tests/asm-perfcounter/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: clean
+
+include ../../mk/toolchain.mk
+
+ASFLAGS = -march=rv32i -mabi=ilp32
+
+all: perfcounter.elf
+
+perfcounter.elf: perfcounter.S
+	$(CROSSCOMPILE)gcc $(ASFLAGS) -o $@ $<
```

Comment:
Should be `$(CROSS_COMPILE)`.

Reply:
Check recent entry for writing proper `Makefile` which deals with C and assembly source. https://github.com/sysprog21/rv32emu/tree/master/tests/chacha20